### ISSUE TITLE
Check file existence before including it in YiiBase::autoload() (no namespace)

### ISSUE THREAD

**Issue by peterngockhoa** [yiisoft/yii#4251]
### What steps will reproduce the problem?
Run test case using PHPUnit 7.5.2+

### What is the expected result?
Do not show warning for include a non-existed file

### What do you get instead?
I run `phpunit --debug api` with "api" is the folder, but it shows warning for include a non-existed file (api.php).
![image](https://user-images.githubusercontent.com/7524282/53934379-31d89500-40d5-11e9-8a13-eb18917e13c8.png)

### Reason
Because PHPUnit 7.5.2+ allows TestSuite to find classes using autoloader: https://github.com/sebastianbergmann/phpunit/pull/3454/commits/22d575d4d35455867a656c06ccaf4d5536390ed1#diff-84cd9176570ec22112cea1681ae0568aL342

### Solution
Use function [stream-resolve-include-path()](http://php.net/manual/en/function.stream-resolve-include-path.php) to check and resolve path before including file at https://github.com/yiisoft/yii/blob/master/framework/YiiBase.php#L463:
```php
				elseif(($classFile=stream_resolve_include_path($className.'.php'))!==false)
				{
					include($classFile);
				}
				else
					return false;
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 1.1.20
| PHP version      | 7.2.13
| PHPUnit version | 7.5.6
| Operating system | Windows 10

**Comment by rob006** [comment]
Duplicates https://github.com/yiisoft/yii/pull/4240

**Comment by peterngockhoa** [comment]
@rob006, although this is duplicated, but how about the solution which I provided? Does it affect performance of Yii? I'm not sure about performance between getting absolute path from `stream_resolve_include_path()` vs using directly `include()`

**Comment by rob006** [comment]
`stream_resolve_include_path()` is not available in PHP 5.1 and 5.2, so we cannot use it.

**Comment by My6UoT9** [comment]
check first php version then do one or the other?

**Comment by rob006** [comment]
But what is the point? If you don't want blind `include` call, set `Yii::$enableIncludePath` to `false`.

**Comment by peterngockhoa** [comment]
> `stream_resolve_include_path()` is not available in PHP 5.1 and 5.2, so we cannot use it.

OK, got it. Thanks!
